src/semantic/color.tokens.json5

```diff
@@ -1122,6 +1180,7 @@
         },
       },
       interaction: {
+        comment: "deprecated",
```

Comment:
Is this intended? we use functional/interaction in the Link

Reply:
Hey Arturo, this actually shouldn't be the case. For the link there is an extra-token that includes all link styles. text&icon.link. …

Reply:
Is this the only place you use it @acstll? And yes, this should be replaced with the link tokens.

Reply:
yep, it's the only place, all clear +1